packages/cli/src/commands/init.ts

```diff
@@ -463,7 +464,7 @@ import { createAppRoute } from "@trigger.dev/nextjs";
 import { client } from "${routePathPrefix}trigger";
 
 // Replace this with your own jobs
```

Comment:
It would be good to remove this comment, from this file and the one in the pages route.

Instead can we add a comment at the top of the jobs/index file: import all your job files here

Reply:
Gotcha